Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2.yml

```diff
@@ -603,6 +759,5 @@ outputs:
 - contextPath: CheckpointFWRule.Hits.Value
   description: Value of rule hits.
 tests:
-- No Test
+- No tests (auto formatted)
 fromversion: 5.0.0
-toversion: 5.4.9
```

Comment:
The `toversion` should stay the same. Congratulations on finding a new SDK bug!
Please open a Jira ticket about this. If a playbook that already exists in content has a `toversion`, the SDK should keep it.